### ISSUE TITLE
Debian/Ubuntu init.d script

### ISSUE THREAD

**Issue by wiktor2200** [danfruehauf/incron#9]
Hello, I needed support for Ubuntu for your great recursive version, so I create workaround.

I've written `init.d` script for Debian/Ubuntu systems. I modified `Makefile` to automatically add this script to `/etc/init.d/incron` and create `service`.

Could we merge it somehow (other branch maybe)? Or should it remain only in my repository and should I rename it something like: `incron-debian-ubuntu`?

Thanks in advance for your reply or some feedback :)

**Comment by wiktor2200** [comment]
Full comparison between your master and my branch: `debian-ubuntu`: <https://github.com/danfruehauf/incron/compare/master...wiktor2200:debian-ubuntu>, so you can take a look :) If you have some ideas how to handle it, I will create pull request.

**Comment by danfruehauf** [comment]
Hi @wiktor2200, thanks for your contributions and feedback.

As for that, I think we can include the init.d file, however the extra commands in the makefile can prove problematic, as this project should be distribution-neutral. In other words, the installation of init.d scripts (or systemd units) should be done by the packager for the specific distribution. And that is something I'm saying as someone who wears the packager hat sometimes (for other packages in Fedora).

So in short, the init.d script can definitely make it in, it might be useful for anyone who wants a reference file to package it for debian/ubuntu, but the actual `makefile` instructions to deploy it will have to be left out. They can be included in debian/ubuntu specific documentation though.
